lib/api_repo.js

```diff
@@ -699,26 +712,36 @@ function newIsDirFunc(dirName) {
 }
 
 /**
- * _verifyRepo confirms that api repo source is available.
+ * _checkRepo confirms that api repo source is available.
  *
- * if repoDir is set, it confirms that the directory exists
+ * if repoDirs is set, it confirms that the directories exists
  *
- * if repoDir is not set, but zipUri is, it downloads the api zip to tmp dir and
- * sets that to repoDir
+ * if repoDirs is not set, but zipUri is, it downloads the api zip to tmp dir
+ * and sets repoDirs to be an array containing that as its single element.
  *
- * if isGoogleApi is `true`, it confirms that repoDir has 'google' subdirectory
+ * if isGoogleApi is `true`, it confirms that each directory in repoDir has
+ * 'google' subdirectory
  * @param {function(?Error)} done The callback.
  */
 ApiRepo.prototype._checkRepo = function _checkRepo(done) {
   var that = this;
-  if (this.repoDir) {
-    var checkDir = newIsDirFunc(this.repoDir);
+  if (this.repoDirs.length) {
+    var checkDirs = this.repoDirs.map(function (repoDir) {
+      return newIsDirFunc(repoDir);
+    });
+    var checkGoogleDirs = []
     if (this.isGoogleApi) {
-      var checkGoogleDir = newIsDirFunc(path.join(that.repoDir, 'google'));
-      async.waterfall([checkDir, checkGoogleDir], done);
-    } else {
-      checkDir(done);
+      checkGoogleDirs = this.repoDirs.map(function (repoDir) {
+        return newIsDirFunc(path.join(repoDir, 'google'));
+      });
     }
+    async.waterfall(checkDirs.concat(checkGoogleDirs), function (err) {
+      if (!err) {
+        done(null);
+      } else {
+        done(err);
+      }
+    });
     return;
   }
```

Comment:
It's not your change, but the structure of this function is really weird -- let's merge the code section into the same part. That means,
```
... = function _checkRepo(done) {
  var that = this;
  function makeTmpDir(next) {
     ...
  }

  if (!_.isEmpty(this.repoDirs)) {
    ...
  } else if (this.zipUrl) {
    ...
  }
}
```

Optional to remove `var that = this`, but you don't have to do in this PR.
# Also, becauze zipUrl should exist if `repoDirs` is empty, I like to omit `this.zipUrl` check.